src/test/java/com/hocs/test/pages/complaints/COMPProgressCase.java

```diff
@@ -106,9 +106,9 @@ public void attemptEscalateCOMPCaseToStage2() throws Exception {
         if (search.checkVisibilityOfEscalationHypertext()) {
             WebElementFacade compCaseRefField = findBy("//a[contains(text(), 'Escalate case')]/parent::td/preceding-sibling::td/a");
             String compCaseRef = compCaseRefField.getText();
-            setSessionVariable("compCaseReference").to(compCaseRef);
+            setSessionVariable("complaintsCaseReference").to(compCaseRef);
```

Comment:
could we change this to indicate which stage this reference is from, stage 1 or stage 2 case?